.github/workflows/ci.yml

```diff
@@ -93,6 +93,22 @@ jobs:
       - run: yarn gulp debug-release ${{ matrix.releaseArgs }}
         if: ${{ inputs.debug_build || matrix.name == 'Android' }}
 
+      - name: repack .deb file with xz compression
```

Comment:
Simply add a comment here about why we are doing this. It will be easier for future developers when they see this code.